features/initialize.feature

```diff
@@ -0,0 +1,32 @@
+Feature: init policies command
+  The ec init policies command should work as expected
+
+  Background:
+    Given a stub cluster running
+    Given stub rekord running
+    Given stub registry running
+    Given stub git daemon running
+    Given stub tuf running
+    Given the environment variable is set "EC_EXPERIMENTAL=1"
+    When ec command is run with "init policies --dest-dir=${TMPDIR}"
+
+  Scenario: success
+    When ec command is run with "test --policy ${TMPDIR} acceptance/examples/empty_input.json -o json"
+    Then the exit status should be 0
+    Then the output should match the snapshot
+
+  Scenario: appstudio success
+    When ec command is run with "test --policy ${TMPDIR} acceptance/examples/empty_input.json -o appstudio"
+    Then the exit status should be 0
+    Then the output should match the snapshot
+
+  Scenario: happy day
+    Given a key pair named "known"
+    Given an image named "acceptance/ec-happy-day"
+    Given a valid image signature of "acceptance/ec-happy-day" image signed by the "known" key
+    Given a valid Rekor entry for image signature of "acceptance/ec-happy-day"
+    Given a valid attestation of "acceptance/ec-happy-day" signed by the "known" key
+    Given a valid Rekor entry for attestation of "acceptance/ec-happy-day"
+    When ec command is run with "validate image --image ${REGISTRY}/acceptance/ec-happy-day --policy ${TMPDIR} --public-key ${known_PUBLIC_KEY} --rekor-url ${REKOR} --strict --show-successes"
```

Comment:
Not sure `${TMPDIR}` is shared between scenarios, I think they're run with different `testing.T`, the snapshot shows that the policy was loaded [from git](https://github.com/enterprise-contract/ec-cli/pull/989/files#diff-916c6b5990f655faf9644c2efb00f8c142433998fe459e42a0d5798a6043531cR92), and I would expect the package to be `policy.release.my_package` not [`main`](https://github.com/enterprise-contract/ec-cli/pull/989/files#diff-916c6b5990f655faf9644c2efb00f8c142433998fe459e42a0d5798a6043531cR61). Looks a bit fishy, interference from another test? Perhaps name everything something like `init-policy` instead of `ec-happy-day` to be sure?